### PR TITLE
adding ETH Montréal

### PR DESCRIPTION
**Pull request by ethmontreal**

### CODE REVIEW

File: src/data/community-events.json [hunk 1]
```diff
@@ -142,6 +142,15 @@
...
     "endDate": "2023-08-27"
+  },
+    {
+    "title": "ETH Montréal",
+    "to": "https://ethmontreal.xyz",
+    "sponsor": null,
+    "location": "Montréal, Canada",
+    "description": "Conference and hackathon with the aim to onboard to Ethereum the students and junior devs from Montréal's world class universities",
+    "startDate": "2023-09-18",
+    "endDate": "2023-09-22"
```
Comment (nhsz) [inline src/data/community-events.json:153]: site says event ends on 21

**Review by nhsz — COMMENTED** [review]
hi, thanks for the contribution, left a comment about the dates

**Review by ethmontreal — COMMENTED** [review]
date corrected, thanks!

**Review by nhsz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**